plutus-contract/plutus-contract.cabal

```diff
@@ -122,6 +122,7 @@ library
     , freer-extras
     , plutus-chain-index-core
     , plutus-ledger
+    , plutus-ledger-api
```

Comment:
This should go in the "Other IOG dependencies" down below.

Reply:
Ah, forgot to revert this. Was trying to fix my breaking HLS.